contractcourt/chain_watcher.go

```diff
@@ -790,12 +806,27 @@ func (c *chainWatcher) handleKnownRemoteState(
 		return false, nil
 	}
 
+	// Create an AnchorResolution for the breached state.
+	anchorRes, err := lnwallet.NewAnchorResolution(
```

Comment:
Don't we create this elsewhere?

Reply:
In the other cases, it's created as part of the `UnilateralCloseSummary` or `LocalForceCloseSummary` in lnwallet

Reply:
We also create one when we `sweepAnchors`.